shopinvader_api_sale/routers/sales.py

```diff
@@ -60,6 +60,7 @@ def get(
         env["shopinvader_api_sale.sales_router.helper"]
         .new({"partner": partner})
         ._get(sale_id)
```

Comment:
Side note, this way of using the abstract model sounds a bit strange to me. I'd prefer pure `@api.model` methods that accept the partner as argument. @sebastienbeau

Reply:
The aim of passing the partner like that, was to be able to have the "self.partner" without needing of add on every method "partner" as first params.
But any better idea are welcome